splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -1163,4 +1115,31 @@ public void upgradeSysColumnsWithUseExtrapolationColumn(TransactionController tc
             SpliceLogUtils.info(LOG, "SYS.SYSCOLUMNS upgraded: added column: USEEXTRAPOLATION.");
         }
     }
+
+    public void removeUnusedBackupTables(TransactionController tc) throws StandardException {
+        SchemaDescriptor sd = getSystemSchemaDescriptor();
+        TableDescriptor td = getTableDescriptor("SYSBACKUPFILESET", sd, tc);
+        if (td != null) {
+            dropTableDescriptor(td, sd, tc);
```

Comment:
Is it enough to just drop the table descriptor if we want to remove the unused backup tables? what about the columns of these tables, and the conglomerate as well as index conglomerates on these tables? Looking at DropTableConstantOperation.executeConstantAction(), dropping a user table seems to have more things to do. Is it similar for dictionary table? Thanks.

Reply:
Good catch!